lib/StaticUniformGrid.js

```diff
@@ -0,0 +1,277 @@
+'use strict';
+var Cesium = require('cesium');
+var AxisAlignedBoundingBox = Cesium.AxisAlignedBoundingBox;
+var Cartesian3 = Cesium.Cartesian3;
+var clone = require('clone');
+
+module.exports = StaticUniformGrid;
+
+/**
+ * A Uniform Grid for unmoving objects.
+ * @alias StaticUniformGrid
+ * @constructor
+ *
+ * @param {Object[]} objects Objects that the grid should store.
+ * @param {Function} aabbFunction A function for checking the axis aligned bounding box of an object in space.
+ *                   This function should expect to take an object and a `min` and `max` Cartesian3. It should update
+ *                   `min` and `max` in place.
+ * @param {Function} cellCheckFunction A function for checking if an object intersects a grid cell, given as 2 Cartesian3s.
+ *                   This function should expect to take an object and the bounding cell's `min` and cell width. It should
+ *                   return `true` if the object intersects with the cell at all and `false` otherwise.
+ * @param {Number} cellWidth, The cell width of the uniform grid.
+ */
+
+var minScratch = new Cartesian3();
+var maxScratch = new Cartesian3();
+
+function StaticUniformGrid(data, cellWidth, objectAABBFunction, cellCheckFunction) {
+
+    /**
+     * The width of a single cell.
+     * @type {Number}
+     */
+    this.cellWidth = cellWidth;
```

Comment:
Most of these properties are immutable, right?  They are set once and then can't be changed later.

Use property getters, https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#property-gettersetters

Inside this class, it is, of course, OK to access the variable directly.